modules/appeals_api/app/controllers/appeals_api/v2/decision_reviews/supplemental_claims_controller.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
+
+require 'json_schema/json_api_missing_attribute'
```

Comment:
We won't ever throw this error if we're using DetailedSchemaErrors

Reply:
Good catch! Thanks!